### PR TITLE
DOCSP-35707: merge gridfs into mongo

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-35707>
**Staging**
- [whats new](https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-35707-merge-gridfs/whats-new/#what-s-new-in-2.0)
- [gridfs](https://preview-mongodbrustagir.gatsbyjs.io/golang/DOCSP-35707-merge-gridfs/fundamentals/gridfs/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working? **Commented out links to fix after apidoc release**
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by jordan-smith721 — APPROVED** [review]
LGTM

File: source/fundamentals/gridfs.txt [hunk 1]
```diff
@@ -87,31 +87,32 @@ Create a GridFS Bucket
...
 an existing bucket on a MongoDB database. To create a ``GridFSBucket`` instance,
-call the ``NewBucket()`` method with a database parameter:
+call the ``GridFSBucket()`` method on a ``Database`` instance, as shown
+in the following code:
 
 .. code-block:: go
 
    db := client.Database("myDB")
-   bucket, err := gridfs.NewBucket(db)
+   bucket, err := db.GridFSBucket()
```
Comment (matthewdale) [inline source/fundamentals/gridfs.txt:96]: This is correct, but may change with https://github.com/mongodb/mongo-go-driver/pull/1533 (which removes the error return value).

Comment (rustagir) [reply source/fundamentals/gridfs.txt:96]: Thanks for pointing that out, I'll wait to merge until that PR is resolved!

File: source/includes/fundamentals/code-snippets/gridfs.go [hunk 2]
```diff
@@ -22,13 +21,14 @@ func main() {
 	defer client.Disconnect(context.TODO())
 
 	// Creates a GridFS bucket
-	bucket, err := gridfs.NewBucket(client.Database("foo"))
+	db := client.Database("myDB")
+	bucket, err := db.GridFSBucket()
```
[the remark this answers is not in the document]
Comment (matthewdale) [reply source/includes/fundamentals/code-snippets/gridfs.go:25]: https://github.com/mongodb/mongo-go-driver/pull/1533 is merged, so we should change this to remove the returned error.

**Pull request merged**